.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "NEOXUiMV8",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isOpen",
-              "parameters": {}
-            },
-            {
-              "name": "hasLabel",
-              "parameters": {
-                "label": "needs triage :mag:"
-              }
-            },
-            {
-              "name": "labelAdded",
-              "parameters": {
-                "label": "In-PR :rocket:"
-              }
-            },
-            {
-              "name": "isPartOfProject",
-              "parameters": {}
-            },
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "isOpen",
-                  "parameters": {}
-                },
-                {
-                  "name": "hasLabel",
-                  "parameters": {
-                    "label": "needs triage :mag:"
-                  }
-                },
-                {
-                  "name": "isAssignedToSomeone",
-                  "parameters": {}
-                },
-                {
-                  "name": "isPartOfProject",
-                  "parameters": {}
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Remove needs triage label",
-        "actions": [
-          {
-            "name": "removeLabel",
-            "parameters": {
-              "label": "needs triage :mag:"
-            }
-          }
-        ]
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "-1mJNNyj7",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isAction",
-              "parameters": {
-                "action": "opened"
-              }
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isPartOfProject",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isAssignedToSomeone",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "name": "hasLabel",
-              "parameters": {
-                "label": "question :grey_question:"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add Question Label ",
```

Comment:
This task didn't get ported over, presumably because it references `isPartOfProject` which is not supported.